now-mainnet.json

```diff
@@ -6,6 +6,7 @@
   "build": {
     "env": {
       "REACT_APP_CHAIN_ID": "1",
+      "REACT_APP_PACKAGE_VERSION": "1.0.0",
```

Comment:
Ah, this we usually inject this in the build process via the package.json, like so: https://github.com/aragon/aragon/blob/master/scripts/build#L5

We might have to update the build script to be a js file or use a CLI json processor (e.g. jq, if it exists in all the different build environments).

Reply:
Ended up updating the `build.sh` into a js file